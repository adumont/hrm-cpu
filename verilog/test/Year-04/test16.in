19 c4
02 fd
36 24
09 0a
32 ec
48 d7
58 ab
48 8a
06 54
04 30
40 b9
02 8f
23 18
40 fa
13 3e
01 eb
24 b9
19 36
28 30
35 be
13 18
06 be
28 d9
21 e0
13 17
19 7c
01 bb
24 0e
21 35
19 b9
08 fe
04 73
18 34
39 64
46 fa
11 b3
31 3d
48 61
12 96
27 07
13 2b
36 ad
44 8c
07 51
36 60
57 8f
