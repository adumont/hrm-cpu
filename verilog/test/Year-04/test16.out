fd
c4
0a
24
d7
ec
8a
ab
30
54
8f
b9
fa
18
eb
3e
36
b9
be
30
be
18
e0
d9
7c
17
0e
bb
b9
35
73
fe
64
34
b3
fa
61
3d
07
96
ad
2b
51
8c
8f
60
